lib/redux/home/myPlot/my_plot_state.dart

```diff
@@ -11,21 +10,28 @@ class MyPlotState {
 
   final LoadingStatus loadingStatus;
   final List<CropEntity> cropList;
+  final CropEntity selectedCrop;
 
-  MyPlotState({@required this.loadingStatus, @required this.cropList});
+  MyPlotState(
+      {@required this.loadingStatus,
+      @required this.cropList,
+      this.selectedCrop});
 
   // We define an initial state
   factory MyPlotState.initial() {
     return new MyPlotState(
-        loadingStatus: LoadingStatus.loading,
-        cropList: new List(0)); // TODO check how to unify this key
+        loadingStatus: LoadingStatus.LOADING, cropList: new List(0));
```

Comment:
I've seen Albert highlight the removal of new keyword. Would it be different if ``cropList: []`` is used instead?

Reply:
The correct update is "List()".
The use of "new" comes from Dart. 
Dart2 enables you to instantiate by context, so no need to add the "new" modifier.
Using it wont do any wrong, but its outside the coding practice of the new dart2, so we should not use it :)